config/config.yml

```diff
@@ -19,7 +19,7 @@ styleGuideExcludes:
 cacheBusterOn: 'true'
 exportDir: export
 publicDir: public
-sourceDir: build
+sourceDir: build/pattern-library
```

Comment:
Currently the only thing inside `build`, but #77 will add a `coverage` directory which shouldn't end up published.